src/fde.h

```diff
@@ -59,6 +59,8 @@ class fde
         write_method = nullptr;
     }
 
+    static void CreateTable() { assert(!Table); Table = static_cast<fde *>(xcalloc(Squid_MaxFD, sizeof(fde))); }
```

Comment:
This method proved to be inlined regardless of 'inline', so left as is.

Reply:
I will rephrase: _If_ you can define this method in fde.cc without causing a lot of trouble, then please define it in fde.cc. By default (i.e. without some exceptional circumstances), this method should not be defined in the header file because the method implementation uses external names (i.e. assert() and xcalloc()) and the method itself is not performance-sensitive.

Reply:
Done.